src/NumericalAlgorithms/Interpolation/Tags.hpp

```diff
@@ -40,8 +40,10 @@ struct InterpolationTargets {
 namespace Tags {
 
 /// Keeps track of which points have been filled with interpolated data.
+template <typename Metavariables>
 struct IndicesOfFilledInterpPoints : db::SimpleTag {
-  using type = std::unordered_set<size_t>;
+  using type = std::unordered_map<typename Metavariables::temporal_id::type,
```

Comment:
`db::item_type` would probably be better (and below)

Reply:
nooooooooooooooo

Reply:
or explicitly using `TimeStepId`?